lib/scripts/eventPage.js

```diff
@@ -148,27 +141,29 @@ function setup() {
     'contexts': ['selection'],
     'id': 'removeFromDifficultyBucket'
   });
+  chrome.contextMenus.create({
+    'title': 'Visual Hints',
```

Comment:
Could you also change the few other menu titles (e.g. "SpeakTheWord", ) that do not conform to the linguistic style of most other titles?